src/appshell/view/navigableappmenumodel.cpp

```diff
@@ -84,6 +84,20 @@ void NavigableAppMenuModel::handleMenuItem(const QString& itemId)
     AppMenuModel::handleMenuItem(itemId);
 }
 
+void NavigableAppMenuModel::openPrevMenu()
+{
+    restoreMUNavigationSystemState();
+    navigate(Qt::Key_Left);
+    activateHighlightedMenu();
+}
+
+void NavigableAppMenuModel::openNextMenu()
+{
+    restoreMUNavigationSystemState();
```

Comment:
It seems like this shouldn't be here
When switching menus, the navigation will blink